jestPolyfills.js

```diff
@@ -0,0 +1 @@
+require('raf/polyfill');
```

Comment:
When updating to React 16.2.0 and running the tests, I was seeing an error in the console that pointed me to [this page](https://reactjs.org/docs/javascript-environment-requirements.html).

This led me to include this file in Jest's [`setupFiles`](https://facebook.github.io/jest/docs/en/configuration.html#setupfiles-array) configuration option to allow for polyfilling `requestAnimationFrame`. This seems to be what [create-react-app does as well](https://github.com/facebook/create-react-app/blob/v1.1.0/packages/react-scripts/scripts/utils/createJestConfig.js#L25).

Let me know if you'd like a different filename or location for this test setup file - can gladly change it.